src/utils/fileUpload.ts

```diff
@@ -0,0 +1,29 @@
+export const uploadFileToStorage = async (
+  file: File,
+  fileExtension: string,
+  assetId: string,
+  schoolId: string,
+  roomId: string,
+): Promise<string> => {
+  const fileToUpload = new File([file], `${assetId}.${fileExtension}`, {
+    type: file.type,
+  });
+
+  const formData = new FormData();
+  formData.append("file", fileToUpload);
+
+  const response = await fetch(
+    `/api/v3/file/upload/${schoolId}/boardnodes/${roomId}`,
```

Comment:
Hard coded urls are danger. If we change the api of the files-storage then we do not check every repository if someone use a api/v3* string. 
(1) In vue a openAPI export for files-storage exists. The execution can be added to the package.json. Bring you also type support. 
(2) The light way. Rename the file to something with "api" and place it to top place like src/*api*.ts.
(3) A config endpoint for files-storage can make it a little bit better. That you only know api/v3/file/config and this give you the details about the api structure. But this need more time to think about it and for now i like if we can avoid it.